packages/flutter/lib/src/material/navigation_rail.dart

```diff
@@ -660,17 +660,38 @@ class _RailDestination extends StatelessWidget {
       selected: selected,
       child: Stack(
         children: <Widget>[
-          Material(
-            type: MaterialType.transparency,
-            child: InkResponse(
-              onTap: onTap,
+          // Material(
+          //   color: Color(0xFFF542DA),
+          //   type: MaterialType.transparency,
+          //   child: InkResponse(
+          //     onTap: onTap,
+          //     onHover: (_) {},
+          //     focusColor: Color(0xFFF542DA),//Color(0xFF4E53BB),
+          //     highlightShape: BoxShape.rectangle,
+          //     borderRadius: BorderRadius.all(Radius.circular(minWidth / 2.0)),
+          //     containedInkWell: true,
+          //     splashColor: colors.primary.withOpacity(0.12),
+          //     hoverColor: colors.primary.withOpacity(0.04),
+          //     child: content,
+          //   ),
+          // ),
+          Padding(
+            padding: EdgeInsets.symmetric(vertical: 8, horizontal: 16),
```

Comment:
These magic numbers here and below are not great. If we need these, we would want to use constants.